starboard/shared/opus/opus_audio_decoder.cc

```diff
@@ -241,9 +245,19 @@ scoped_refptr<OpusAudioDecoder::DecodedAudio> OpusAudioDecoder::Read(
 void OpusAudioDecoder::Reset() {
   SB_DCHECK(BelongsToCurrentThread());
 
-  TeardownCodec();
-  InitializeCodec();
+  if (is_valid()) {
+    int error = opus_multistream_decoder_ctl(decoder_, OPUS_RESET_STATE);
+    if (error != OPUS_OK) {
+      SB_LOG(ERROR) << "Failed to reset decoder with error: "
```

Comment:
I think we can be a bit more descriptive here in case we don't know what file this is coming from, so "decoder" -> "OpusAudioDecoder"

Reply:
Fixed.